### ISSUE TITLE
In light theme, focus on dismis button in search results is not visible

### ISSUE THREAD

**Issue by ramya-rao-a** [Microsoft/vscode#58545]
Steps to Reproduce:

1. Use search, Ctrl+Down arrow to move to results
2. Use Down/Up arrow to traverse the list
3. Use tab once to move focus to the dismiss button on the current search result

Observe that the outline on the dismiss button is not visible.

This happens in both the default light themes

cc @misolori

**Comment by misolori** [comment]
This appears to happen because our selected color and our focus color are very similar and are very hard to see. Will need to invert the focus color here on selected rows.

![image](https://user-images.githubusercontent.com/35271042/45450195-4d199900-b68c-11e8-9ac1-df0435bdc845.png)

**Comment by misolori** [comment]
Fixed this so that we use `list.ActiveSelectionForeground` for the outline color on selected actions

![image](https://user-images.githubusercontent.com/35271042/45452871-fadc7600-b693-11e8-962d-bdef22fe55cb.png)


![image](https://user-images.githubusercontent.com/35271042/45452858-f1eba480-b693-11e8-838f-a52a5cc16e28.png)

![image](https://user-images.githubusercontent.com/35271042/45452836-ea2c0000-b693-11e8-9f3a-d1e5be7c68ae.png)
